### PR TITLE
[WIP] Image MicroService

### PR DESCRIPTION
**Pull request by mattwr18**
- Maybe this isn't helpful at all, but since no one else has opened up a
WIP PR, I though I would try to learn a bit before the holidays. I hope
we can use this as a starting point to start a discussion about this.

- Following tutorial https://malcoded.com/posts/nodejs-image-resize-express-sharp/

### CODE REVIEW

File: image-micro-service/src/server.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
+const express = require('express')
+const resize = require('./resize')
+const server = express()
+
+server.get('/', (req, res) => {
```
Comment (roschaefer) [inline image-micro-service/src/server.js]: This looks great! Maybe we can merge this into the backend in order to quickly merge the responsive image feature?

**Comment by mattwr18** [comment]
should I just close this then @Human-Connection/hc-dev-team ??

**Comment by Tirokk** [comment]
We should discuss it in the team with all other possible solutions together and make a team decision at all. Or? @mattwr18 @roschaefer

File: image-micro-service/src/server.js (hunk 1 above)
Comment (mattwr18) [reply image-micro-service/src/server.js]: how would this work with what we have in the backend? 
I'm looking at this as a potential solution for https://github.com/Human-Connection/Human-Connection/issues/1922
but I don't think we would use `server.get('/')`
I couldn't easily figure out what it should be... I think maybe `server.get('/api/')`, oder? @roschaefer

**Comment by mattwr18** [comment]
we are about to be in a position to re-evaluate this `PR` and see if we want to go in this direction(resize on the fly), or load multiple images to `spaces`

**Comment by mattwr18** [comment]
closing in favor of [imgproxy](https://github.com/imgproxy/imgproxy)

**Pull request closed without merging**